osquery/tables/networking/windows/process_open_sockets.cpp

```diff
@@ -0,0 +1,270 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <osquery/core.h>
+#include <osquery/core/conversions.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "win_sockets.h"
+
+namespace osquery {
+namespace tables {
+
+WinSockets::WinSockets() : status_(Status(0, "OK")) {
```

Comment:
nit, is this initialization needed? The constructor for `Status` does the same thing more or less.